utils/release.py

```diff
@@ -12,41 +12,84 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+Utility that prepares the repository for releases (or patches) by updating all versions in the relevant places. It
+also performs some post-release cleanup, by updating the links in the main README to respective model doc pages (from
```

Comment:
when I read this, I am thinking this is only for the `post-release`. But after reading later stuff below, it seems to me the changes will (and should) be on the release branch and the main branch (after the release).

I am not strong for a change here, but if you happen to have a better wording, it would be nice.